test/machinery_test.exs

```diff
@@ -4,70 +4,99 @@ defmodule MachineryTest do
 
   alias MachineryTest.Helper
   alias MachineryTest.TestStruct
+  alias MachineryTest.TestDefaultFieldStruct
```

Comment:
The alias `MachineryTest.TestDefaultFieldStruct` is not alphabetically ordered among its group.